Source/Private/ASIGListAdapterBasedDataSource.mm

```diff
@@ -351,7 +351,7 @@ + (void)configureUpdater:(id<IGListUpdatingDelegate>)updater
 {
   // Cast to NSObject will be removed after https://github.com/Instagram/IGListKit/pull/435
   if ([(id<NSObject>)updater isKindOfClass:[IGListAdapterUpdater class]]) {
-    [(IGListAdapterUpdater *)updater setAllowsBackgroundReloading:NO];
+   // [(IGListAdapterUpdater *)updater setAllowsBackgroundReloading:NO];
```

Comment:
Shouldn't be commented, bug is still relevant, dataSource won't update properly using Texture+IGListKit when `self.setAllowsBackgroundReloading == true, collectionView.window == nil`

Reply:
seems like no longer valid on the latest IGListKit